src/cheri_pc_access.sail

```diff
@@ -64,7 +64,7 @@
 
 val get_arch_pc : unit -> xlenbits effect {rreg}
 function get_arch_pc () =
-  PC - getCapBaseBits(PCC)
+  if have_pcc_relocation() then PC - getCapBaseBits(PCC) else PC
 
 /* FIXME: The asymmetry between get_next_pc and set_next_pc where one
```

Comment:
Do you want to update this FIXME comment to say it only applies if have_pcc_relocation() is true?

Reply:
Updated